examples/imdb_bidirectional_lstm.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(1337)  # for reproducibility
+
+from keras.preprocessing import sequence
+from keras.optimizers import SGD, RMSprop, Adagrad
+from keras.utils import np_utils
+from keras.models import Sequential, Graph
+from keras.layers.core import Dense, Dropout
+from keras.layers.embeddings import Embedding
+from keras.layers.recurrent import LSTM, GRU
+from keras.datasets import imdb
+
+from sklearn.metrics import accuracy_score as accuracy
+
+'''
+    Train a Bidirectional LSTM on the IMDB sentiment classification task.
+    The dataset is actually too small for LSTM to be of any advantage
+    compared to simpler, much faster methods such as TF-IDF+LogReg.
+    Notes:
+    - RNNs are tricky. Choice of batch size is important,
+    choice of loss and optimizer is critical, etc.
+    Some configurations won't converge.
+    - LSTM loss decrease patterns during training can be quite different
+    from what you see with CNNs/MLPs/etc.
+    GPU command:
+        THEANO_FLAGS=mode=FAST_RUN,device=gpu,floatX=float32 python imdb_lstm.py
```

Comment:
A description that is customized to this specific example would be preferable, specifically point out what there is to know about bidirectional networks and what should be expected of them.